macro/run_simScript.py

```diff
@@ -115,6 +118,10 @@
 if options.genie:    simEngine = "Genie"
 if options.nuradio:  simEngine = "nuRadiography"
 if options.ntuple:   simEngine = "Ntuple"
+if options.alpaca:
+    options.inputFile= False
+    CaloDesign = "2"
```

Comment:
This behaviour might be very surprising to the user!

Would suggest removing it. For studies ALP→digamma one would of course have to enable split-cal (although I expect it would probably  become default sooner or later anyway?

Also, are you sure CaloDesign takes a string and not an int?

Reply:
apparently it takes integer because it does not work.
So, deleting it would be better.